rcl_interfaces/msg/RclHeader.msg

```diff
@@ -0,0 +1,5 @@
+builtin_interfaces/Time stamp
+# A fully qualified ROS node name
+string sender_node_name
```

Comment:
I think you could just simplify it to `node_name` or juse `node`.

you're going to be addressing it as 
`msg.(rcl_)header.stamp` and `msg.(rcl_)header.node`

Other suggestions are `source` or `authority` but I think this is just saying what node at what time so `node` and `stamp` seem best. The name is implicit.